code/_onclick/item_attack.dm

```diff
@@ -77,6 +77,10 @@
 
 	log_combat(user, M, "attacked", src.name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYPE: [uppertext(damtype)])")
 	add_fingerprint(user)
+	//HONK - start
+	if(smelted_material)
+		smelted_material.reaction_mob(M, TOUCH, 30)
+	//HONK - end

```

Comment:
guess I should probably check for caps in the regex I use to search for these comments, huh

Reply:
also does this imply that smelted materials do 30 damage per hit?

Reply:
you want me to make it lowercase

Reply:
Naw thats just how much is transferred if you splash, spray or smoke the reagent on someone

Reply:
thats just a step for the future when i add more fluff you'll get hurt because of molten metal